src/main/java/org/oscarehr/common/model/Relationships.java

```diff
@@ -143,11 +143,11 @@ public void setNotes(String notes) {
         this.notes = notes;
     }
 
-    public String getDeleted() {
+    public boolean getDeleted() {
         return deleted;
     }
 
-    public void setDeleted(String deleted) {
```

Comment:
That's a pretty big change going from a boolean to a string - seems intentional.   Is it from an upstream commit? Feels like a rebase issue and changing the method isn't the solution maybe?

Reply:
The reason I made this change was because upstream was using a boolean.

This is what Magenta's main, alpaca, and bullfrog has ([link](https://github.com/MagentaHealth/Open-O/blob/develop/bullfrog/src/main/java/org/oscarehr/common/model/Relationships.java)):
```java
public void setDeleted(boolean deleted) {
    this.deleted = deleted;
}
```